### ISSUE TITLE
NETW-2600 bogus result

### ISSUE THREAD

**Issue by melak** [CISOfy/lynis#757]
**Describe the bug**
The test concludes
```
[+] Networking
------------------------------------
- Checking IPv6 configuration [ ENABLED ]
Configuration method [ AUTO ]
IPv6 only [ NO ]
```
when IPv6 is completely disabled:
```
# cat /etc/sysctl.d/80-ipv6.conf
net.ipv6.conf.all.disable_ipv6=1
net.ipv6.conf.default.disable_ipv6=1
net.ipv6.conf.eth0.disable_ipv6=1
net.ipv6.conf.lo.disable_ipv6=1
# ip r g ::1
RTNETLINK answers: Network is unreachable
# ip -6 ro ls
#            
```

**Version**
 - Ubuntu 18.04
 - Lynis 2.7.5-241-g0088593

**Expected behavior**
Lynis detects that IPv6 is disabled

**Output**
```
2019-09-11 04:30:33 Performing test ID NETW-2600 (Checking IPv6 configuration)
2019-09-11 04:30:35 Result: IPV6 mode is auto
2019-09-11 04:30:35 Result: IPv6 only configuration: NO
```
**Additional context**

**Comment by mboelen** [comment]
Yes, they are two different things. In your case, IPv6 support is active/loaded (the kernel module is loaded) AND there are some things related to IPv6 configured (so that is why the configuration is 'enabled').

And yes, although you may have defined that you don't want to have IPv6 enabled on your interfaces, the protocol support is there. I understand that the used wording can be interpreted in different ways. I suggest to disable the kernel module, so nothing is loaded and therefore the configuration is disabled as well.
